vulnerable_people_form/form_pages/cookies.py

```diff
@@ -5,4 +5,4 @@
 
 @form.route("/cookies", methods=["GET"])
 def get_cookies():
-    return render_template_with_title("cookies.html", back_url=dynamic_back_url())
+    return render_template_with_title("cookies.html", previous_url=dynamic_back_url())
```

Comment:
The `cookies.html` template appears to refer to back_url
`<a class="govuk-link cookie-settings__prev-page" href={{back_url}}>Go back to the page you were looking at</a>`

Take a look and see if anything needs amending here...

Reply:
@danwilliams ?

Reply:
Good spot! Corrected 🙂